src/api/post/asyncGetPosts.js

```diff
@@ -5,16 +5,18 @@ const asyncGetPosts = async (
   cursorId = POST_LISTS.DEFAULT_CURSOR_ID,
   {
     keywordId,
+    order = POST_LISTS.DEFAULT_ORDER,
     includedKeyword = POST_LISTS.DEFAULT_INCLUDED_KEYWORD,
     excludedKeyword = POST_LISTS.DEFAULT_EXCLUDED_KEYWORD,
+    isAd = POST_LISTS.DEFAULT_IS_AD,
     limit = POST_LISTS.DEFAULT_LIMIT,
   }
 ) => {
   const includedKeywordParams = includedKeyword.length === 0 ? "" : includedKeyword.join();
   const excludedKeywordParams = excludedKeyword.length === 0 ? "" : excludedKeyword.join();
   const fetchInfo = {
     url: `${BASE_URL}/posts/${keywordId}`,
-    params: `?includedKeyword=${includedKeywordParams}&excludedKeyword=${excludedKeywordParams}&limit=${limit}&cursorId=${cursorId}`,
+    params: `?order=${order}&includedKeyword=${includedKeywordParams}&excludedKeyword=${excludedKeywordParams}&isAd=${isAd}$limit=${limit}&cursorId=${cursorId}`,
```

Comment:
```suggestion
    params: `?order=${order}&includedKeyword=${includedKeywordParams}&excludedKeyword=${excludedKeywordParams}&isAd=${isAd}&limit=${limit}&cursorId=${cursorId}`,
```
진태님! `${isAd}` 뒤에 `&` 가 아니라 `$`로 되어있어서 수정 부탁드립니다!

Reply:
오 발견해주셔서 감사합니다! 자칫 테스트 때 오류가 생겼었겠네요, 수정했습니다. :)